### ISSUE TITLE
After installation admin login page empty

### ISSUE THREAD

**Issue by meteyalciner** [Polygant/OpenCEX#12]
I install exchange on ubuntu 20.04 on local . installation finished successfuly but i try to login admin panel it shows empty page

**Comment by mohamedHusseinDigipalms** [comment]
hi @meteyalciner, have you found a solution yet, I'm experiencing the same problem

**Comment by meteyalciner** [comment]
yes I found , I installed ubuntu 22.04 fresh install and install opencex egain all is fine

**Comment by mohamedHusseinDigipalms** [comment]
Thank you will try it out

**Comment by linkedshot** [comment]
Hello. did you install it this way?

mkdir /app ; 
cd /app/ || exit ; 
git clone https://github.com/Polygant/OpenCEX.git ./deploy ;
cd deploy ; 
chmod +x opencex.sh ;
./opencex.sh 2>&1 | tee /tmp/install.txt

Did you have any problems with the domain or ssl?
